.github/workflows/ci.yml

```diff
@@ -282,6 +282,56 @@ jobs:
         # npm b/c Yarn 4 doesn't work in Node 12
         run: cd packages/ses && npm run test:platform-compatibility
 
+  hermes-test:
+    name: hermes-test
+
+    # begin macro
+
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [20.x]
+        platform: [ubuntu-latest]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      # without this, setup-node errors on mismatched yarn versions
+      - run: corepack enable
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: yarn
+
+      - name: Echo node version
+        run: node --version
+
+      - name: Install dependencies
+        run: yarn install --immutable
+
+      # end macro
+
+      - name: 'build'
+        run: yarn run build
+
+      - name: Fetch Hermes CLI for Linux tar file
+        run: cd packages/ses && mkdir hermes-cli-linux && cd hermes-cli-linux && wget https://github.com/facebook/hermes/releases/download/v0.12.0/hermes-cli-linux-v0.12.0.tar.gz
```

Comment:
follow-up: [building and running Hermes](https://github.com/facebook/hermes/blob/main/doc/BuildingAndRunning.md) on MM mobile RN ver <s>.71</s> [0.72](https://github.com/MetaMask/metamask-mobile/pull/9280)

[hermes-2024-04-29-RNv0.72.14-3815fec63d1a6667ca3195160d6e12fee6a0d8d5: Removing API usage not applicable on iOS (stat and fstat) in libhermes](https://github.com/facebook/hermes/releases/tag/hermes-2024-04-29-RNv0.72.14-3815fec63d1a6667ca3195160d6e12fee6a0d8d5)

Reply:
i.e.

```bash
git clone https://github.com/facebook/hermes.git
git checkout hermes-2024-04-29-RNv0.72.14-3815fec63d1a6667ca3195160d6e12fee6a0d8d5
cmake -S hermes -B build -G Ninja
```

Reply:
or npm install (https://www.npmjs.com/package/hermes-engine-cli?activeTab=versions) more consistent

Reply:
`curl` failing due to not following redirects, find appropriate flag as alternative to `wget`